src/core/Akka.Remote.Tests.Performance/Transports/RemoteMessagingThroughputSpecBase.cs

```diff
@@ -84,9 +85,15 @@ public void Setup(BenchmarkContext context)
 
             var system1EchoActorPath = new RootActorPath(system1Address) / "user" / "echo";
             var system2RemoteActorPath = new RootActorPath(system2Address) / "user" / "benchmark";
-            _remoteReceiver =
-                System1.ActorSelection(system2RemoteActorPath).Ask<ActorIdentity>(new Identify(null), TimeSpan.FromSeconds(2)).Result.Subject;
-            _remoteEcho =
+
+            System2.ActorSelection(system1EchoActorPath).Ask<string>("foo").ContinueWith(tr =>
+            {
+                Console.WriteLine("Received foo");
+            });
+            var remoteTask1 = System1.ActorSelection(system2RemoteActorPath).Ask<ActorIdentity>(new Identify(null));
+            remoteTask1.Wait();
+            _remoteReceiver = remoteTask1.Result.Subject;
+           _remoteEcho =
                 System2.ActorSelection(system1EchoActorPath)
                     .Ask<ActorIdentity>(new Identify(null), TimeSpan.FromSeconds(2))
                     .Result.Subject;
```

Comment:
`System2.ActorSelection(system1EchoActorPath).ResolveOne().Result.Subject` would be more idiomatic.
Might not matter in the test but I've seen the above pop up a few times before.